src/main/java/nl/tudelft/dnainator/graph/impl/Neo4jGraph.java

```diff
@@ -88,39 +74,23 @@ public void run() {
 			service.schema().getIndexes().forEach(e -> e.drop());
 
 			service.schema().constraintFor(nodeLabel)
-			.assertPropertyIsUnique(ID)
+			.assertPropertyIsUnique(PropertyTypes.ID.name())
```

Comment:
Can't you use ID.name() anymore because you still import the property type statically?